pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: "Migrating from GOV.UK PaaS"
+---
+
+# Migrating from GOV.UK PaaS
+
+## Last updated
+
+- 1st November 2022 - first published
+
+## Government cloud hosting strategy
+
+The Government Digital Service (GDS) has provided GOV.UK PaaS since 2015, supporting and providing a public cloud platform for departments using a shared hosting and responsibility model. 
```

Comment:
@LeePorte @wryobservations the digital marketplace came onboard in july 2016 so I'm inclined to change this to 2016

Reply:
I don't have a strong view, altho we've always used the alpha as a measure of time in other write ups on this.